react/components/RenderProvider.tsx

```diff
@@ -962,7 +959,7 @@ class RenderProvider extends Component<Props, RenderProviderState> {
 
   private prefetchPages = () => {
     if (this.prefetchRoutes.size > 0) {
-      setTimeout(this.execPrefetchPages, 20 * 1000)
+      setTimeout(this.execPrefetchPages, 6 * 1000)
```

Comment:
Why use `setTimeout` instead of idle event from window?

Reply:
@Klynger good question, but this is just an adjustment that has hitched a ride on this PR. using the idle event might be good for another PR